.github/workflows/publish.yml

```
@@ -207,6 +207,60 @@ jobs:
           name: vector-${{ env.VECTOR_VERSION }}-armv7-unknown-linux-musleabihf
           path: target/artifacts/vector*
 
+  build-arm-unknown-linux-gnueabi-packages:
+    name: Build Vector for arm-unknown-linux-gnueabi (.tar.gz)
+    runs-on: [ linux, release-builder ]
```

```suggestion
    runs-on: [ linux, release-builder ]
    timeout-minutes: 60
```

We started adding these job timeouts proactively to avoid jobs inadvertently hanging for 6 hours by default. Is 60 minutes sufficient do you think? That's the timeout for the armv7 builds.